api/events/lambda_function.py

```diff
@@ -41,7 +41,7 @@ def lambda_handler(event, _):
         # catch all the various forms of ARN from the media services
         arn_expr = parse('$..arn|aRN|resource-arn|channel_arn|flowArn|PlaybackConfigurationArn|resourceArn')
```

Comment:
We need to add "multiplex_arn" to this list of expressions we're checking for, that way we catch it and save that event even if it's not an alert.